### PR TITLE
Split panels

### PR DESCRIPTION
**Pull request by roperzh**
OK to merge.

Short review.

Summary of work (include Asana links if any):

- [x] Show a simple horizontal gray indicator that tracks mouseposition when dragging, then it only commits & resizes upon mouseup https://app.asana.com/0/548868462189817/505957877019451

Regressions to look for:

- Anything related to resizing or stage and timeline layouts

Notes:

We discussed showing a haiku while the stage is resizing, but honestly, the resize is so fast that the haiku just flashes away. We could add an artificial delay to show the haiku but feels like unnecessary. In any case, if @taylorpoe or @zackbrown feels like we should include it, it's a 0 effort task.

Completed checkin tasks:

- [x] Did manual testing of interrelated functionality
- [x] Ran `$ yarn lint-all` and fixed all formatting issues
- [x] Ran `$ yarn test-all` and all tests passed
- [ ] Did the [E2E checklist](https://haiku.quip.com/dGqeAEVSWdmg)
- [x] Expanded the [QA checklist](https://haiku.quip.com/rqwsAPsCGxqT)
- [ ] Wrote an automated test covering new functionality

### CODE REVIEW

File: packages/haiku-creator/package.json [hunk 1]
```diff
@@ -58,7 +58,6 @@
     "react-motion": "0.4.7",
     "react-onclickoutside": "^6.4.0",
     "react-popover": "^0.4.18",
-    "react-split-pane": "0.1.57",
```
Comment (stristr) [inline packages/haiku-creator/package.json:61]: Adios!

Comment (roperzh) [reply packages/haiku-creator/package.json:61]: 👋

**Comment by taylorpoe** [comment]
Looks good. Let's **not** add an artificial delay in order to show a loader. More loaders we show the more the users feel like we're making them wait.

**Comment by tsortwehttam** [comment]
LGTM. Merging.

**Pull request merged**